x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/RuleQueryBuilder.java

```diff
@@ -57,14 +56,9 @@ public class RuleQueryBuilder extends AbstractQueryBuilder<RuleQueryBuilder> {
     public static final String NAME = "rule_query";
 
     private static final ParseField RULESET_ID_FIELD = new ParseField("ruleset_id");
-    private static final ParseField MATCH_CRITERIA_FIELD = new ParseField("match_criteria");
+    protected static final ParseField MATCH_CRITERIA_FIELD = new ParseField("match_criteria");
```

Comment:
If this is not aimed at subclasses but for package visibility, we could use the following:

```suggestion
   static final ParseField MATCH_CRITERIA_FIELD = new ParseField("match_criteria");
```